vllm/engine/async_llm_engine.py

```diff
@@ -117,7 +117,8 @@ def process_request_output(self,
         self._request_streams[request_id].put(request_output)
         if request_output.finished:
             if verbose:
-                logger.info("Finished request %s.", request_id)
+                logger_data = {"request_id": request_id}
```

Comment:
```suggestion
                logger_data = {"event": "request-completed", "request_id": request_id}
```

Reply:
This seems like a defendable use of logger_data; an enterprise might want to have a sense of how frequently requests encounter complete successfully.